src/Components/Body/Chat/Chat.tsx

```diff
@@ -0,0 +1,44 @@
+import ChatLeftFold from "./LeftFold";
+import ChatRightFold from "./RightFold";
+import "./style.css";
+import { useState } from "react";
+import {useAllUsers} from '../../../hooks/useAllUsers'
+import {Spinner} from '../../Spinner/Spinner'
+import { ErrorState } from "../../Error/ErrorState";
+import { ChatMessageType, ChatListDataType } from "../../../Types/types";
+export function Chat() {
+  const {allUserDetails: chatList,loading,error, updateQuery} = useAllUsers()
+  const [selectedChat, setSelectedChat] = useState<number>(0);
```

Comment:
Initial value in this state should be undefined.

Initially no chat will be selected. we can show empty state in the right pane.

Reply:
`<ChatRightFold chatData={chatList?.[selectedChat]} onNewMessage={handleNewMessage} />`
 Initially undefined will be passed to the rightPane